clang/test/SemaSYCL/delayed-parsing.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s
+// RUN: %clang_cc1 -std=c++1y -verify -fsyntax-only %s -DDELAYED_TEMPLATE_PARSING -fdelayed-template-parsing
+// RUN: %clang %s -DDELAYED_TEMPLATE_PARSING -fsycl -c
+// REQUIRES: system-windows
```

Comment:
Why is this required?  This test shouldn't depend on the platform.

Reply:
So what I was trying to do is:
1.  create a test case that needs -fdelayed-parsing to compile cleanly
2.  demo that with -fdelayed-parsing it does compile cleanly
These are the first two RUN cases.   These are not platform-dependent
3.  was to demo that the clang driver passes the required options to the compiler and compiles it cleanly.
For this, I needed the platform dependency, since the Linux compiler does not need the delayed-parsing option.  I could not figure out how to use triples with the clang driver.

Is there a platform-neutral way to demo 3?  I could use that instead.  Thanks.